backend/app/api/v1/profile.py

```
@@ -0,0 +1,110 @@
+from app.models.database.supabase import UserProfileEdit
+
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+security = HTTPBearer()
+
+
+async def get_user(
+    credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)